src/site/content/en/fast/choose-the-right-image-format/index.md

```diff
@@ -186,17 +187,25 @@ In terms of older image formats, consider the following:
     * What about GIF? GIF limits the color palette to at most 256 colors,
       and creates significantly larger file sizes than `<video>` elements. See
       [Replace animated GIFs with video](/replace-gifs-with-videos/).
-1. **Do you need to preserve fine detail with highest resolution? Use PNG.**
+1. **Do you need to preserve fine detail with highest resolution? Use PNG or lossless WebP.**
     * PNG does not apply any lossy compression algorithms beyond the choice of the size of the color palette.
     As a result, it will produce the highest quality image,
     but at a cost of significantly higher filesize than other formats. Use judiciously.
+    * WebP has a lossless encoding mode that is usually more efficient than PNG. Use lossless WebP when possible.
     * If the image asset contains imagery composed of geometric shapes, consider converting it to a vector (SVG) format!
     * If the image asset contains text, stop and reconsider. Text in images is not selectable, searchable, or "zoomable".
     If you need to convey a custom look (for branding or other reasons), use a web font instead.
-1. **Are you optimizing a photo, screenshot, or a similar image asset? Use JPEG.**
+1. **Are you optimizing a photo, screenshot, or a similar image asset? Use JPEG or lossy WebP.**
     * JPEG uses a combination of lossy and lossless optimization to reduce filesize of the image asset. Try several JPEG quality levels to find the best quality versus filesize tradeoff for your asset.
+    * Lossy WebP may be an acceptable JPEG alternative, but be aware that WebP's lossy mode discards some chroma information in order to achieve smaller images. This means that select colors may not be the same as an equivalent JPEG.
```

Comment:
Are we not ready to recommend AVIF yet? Its size improvements over JPEG (and even WebP) are impressive.

Appreciate it's browser support is more limited, but with [upcoming Safari support](https://twitter.com/jensimmons/status/1547693585922789381?s=20&t=qpSRzRCrAfpeOBqjdbhwXw) that's going to be greatly improved.

I have heard stories of slower decode times for larger images though, but not sure how true that is or if just anecdotal. Not sure if you have any more info on this?

If adding AVIF then should update [this table too](https://web.dev/choose-the-right-image-format/#features-of-different-raster-image-formats).